test/Test/Toml/Parsing/Examples.hs

```diff
@@ -0,0 +1,26 @@
+module Test.Toml.Parsing.Examples where
+
+import Data.Either (isRight)
+import Test.Tasty.Hspec (Spec, describe, it, shouldReturn)
+import Toml.Parser (parse)
+
+import qualified Data.Text.IO as TIO
+
+spec_Examples :: Spec
+spec_Examples = describe "Can parse official TOML examples" $
+    mapM_ example files
+
+example :: FilePath -> Spec
+example file = it ("can parse file " ++ file) $
+      (isRight . parse <$> TIO.readFile ("test/examples/" ++ file))
+      `shouldReturn` True
+
+files :: [FilePath]
+files =
+    [ "fruit.toml"
```

Comment:
I propose to use `listDirectory` function inside tests to avoid hardcoding of test cases:

* https://hackage.haskell.org/package/directory-1.3.4.0/docs/System-Directory.html#v:listDirectory

You can use `runIO` function from `hspec` to run `IO` actions inside `Spec` monad:

* https://hackage.haskell.org/package/hspec-2.7.1/docs/Test-Hspec.html#v:runIO

Reply:
I looked into `runIO` when I was figuring out how to read files in a `Spec` but then I realized `it` can handle it just fine, I think I'll leave it like this.

Reply:
Never mind, I guess it's necessary if I want to use `listDirectory`. On it.